src/main/java/gr/uom/java/xmi/diff/MoveCodeRefactoring.java

```diff
@@ -116,7 +116,11 @@ public List<CodeRange> rightSide() {
 
 	@Override
 	public RefactoringType getRefactoringType() {
-		return RefactoringType.MOVE_CODE;
+        if (bodyMapper.getOperation2().hasSetUpAnnotation() || bodyMapper.getOperation2().getName().equals("setUp") || bodyMapper.getOperation2().getName().equals("prepare")) {
```

Comment:
@victorgveloso 
It is important to check if the `moveType` is `MOVE_TO_ADDED`, because the fixture might be pre-existing and still move code to it, but in that case it is not Extract.
It is possible to extract a `tearDown` fixture too.